### DISCUSSION TITLE
Does this library work with regular ink printers?

### DISCUSSION THREAD

**Discussion by nedigan** [lukevp/ESC-POS-.NET#238] [Q&A]
I could be asking a stupid question so please forgive me. However, I would like to test this first on my printer at home before having to purchase a thermal espos printer. Will this library still work?

Thanks

**Comment by igorocampos** [comment]
No it doesn't. Regular ink printers have their own drivers and don't accept ESC/POS commands.
Also be sure that the thermal printer you are about to buy supports ESC/POS commands, otherwise, it won't work either.
